code/modules/mob/living/simple_animal/bot/mulebot.dm

```diff
@@ -362,13 +362,10 @@ var/global/mulebot_count = 0
 	update_icon()
 
 /mob/living/simple_animal/bot/mulebot/proc/load_mob(mob/living/M)
-	if(M.buckled)
-		return 0
-	passenger = M
-	load = M
-	can_buckle = 1
-	buckle_mob(M)
-	can_buckle = 0
+	if(buckle_mob(M))
+		passenger = M
+		load = M
+		can_buckle = FALSE
 	return 1

```

Comment:
only return 1 if buckle_mob succeeds. there's an if(!load_mob(M)) used in the proc above.